ReSharper.FSharp/src/FSharp.Common/src/Util/FSharpXmlDocService.fs

```diff
@@ -13,9 +17,26 @@ open JetBrains.ReSharper.Psi.XmlIndex
 open JetBrains.UI.RichText
 open JetBrains.Util
 
+[<ShellComponent>]
+type FSharpXmlDocHtmlPresenter(theming: ITheming, factory: XmlDocSectionFactory) =
+    inherit XmlDocHtmlPresenter(theming, factory)
+
+    let crefManager =
+        { new CrefManager() with
+            member x.Process(cref, _, _, _, _) = XmlDocPresenterUtil.ProcessCref(cref)
+            member x.Create _ = null }
+
+    member x.Run(node: XmlNode) =
+        let result = RichText()
+        let declaredElement = null.As<DeclaredElementInstance>()
```

Comment:
Do we really need this safe cast here?

Reply:
this method has a deprecated overload with DeclaredElement

Reply:
OK. Why the cast is needed? Why not specify the type explicitly?

Reply:
Also, can we achieve what we need without using a deprecated method? Please consider discussing adding a needed method overload with subsystem owners.

Reply:
we are using the actual method, but there is a deprecated overload with DeclaredElement instead of DeclaredElementInstance

Reply:
OK, I see thanks. Still, could you just annotate the type instead of the conversion?